frontend/src/Header/Header.css

```diff
@@ -1,12 +1,5 @@
 header {
-    grid-area: header;
-    background-color: var(--standardBackgroundColor);
-    color: var(--standardFondColor);
-    display: grid;
-    grid-template-columns: 4fr 1fr;
-    align-items: center;
-    border-radius: 5px 5px 0 0;
-    border-bottom: 3px solid var(--highlightColor);
+
```

Comment:
check sonar isse, you could delete this here

Reply:
ahh, yes you're right. Thanks